images/php-laravel/docker-entrypoint.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+/usr/bin/supervisord
```

Comment:
No need to have this docker-entrypoint.sh script. I think you can just make the entrypoint be supervisord.